_articles/organizations/user-seats.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: article
+title: User Seat Management for your Organization
+categories: [organizations]
+featured: true
+popular: false
+tags: []
+---
+
+## Add User Seats
+
+You can add user seats at any time by logging into the **Web Vault** (https://vault.bitwarden.com) and go to **Settings** &rarr; **Organizations** &rarr; {YOUR ORG NAME} &rarr; **Settings** sub-tab (Gears Icon) &rarr; **Subscription**. There you can locate the **User Seats** section and select the **Add Seats** button.
```

Comment:
Also use the icon plugin helper syntax for the gears icon, then you don't need to put it in parens. Ex:

```
{% icon fa-cog %} **Settings**
```